ivy_tests/test_ivy/test_misc/test_array.py

```diff
@@ -995,6 +1003,10 @@ def test_array__truediv__(
     on_device,
 ):
     dtype, x = dtype_and_x
+
+    # bfloat16 is not supported by numpy
+    assume(not ("bfloat16" in dtype))
```

Comment:
We shouldn't be skipping specific dtypes, this ideally shouldn't be generated for NumPy backend anyways.

Reply:
My bad, I'll look into this, `asarray` is called with torch backend somewhere and torch.as_tensor really said `TypeError: can't convert np.ndarray of type bfloat16.` :skull: